lib/Solver/Z3Solver.h

```diff
@@ -27,7 +27,7 @@ class Z3Solver : public Solver {
   /// setCoreSolverTimeout - Set constraint solver timeout delay to the given
   /// value; 0
   /// is off.
-  virtual void setCoreSolverTimeout(time::Span timeout);
+  virtual void setCoreSolverTimeout(time::Span timeout) override;
```

Comment:
If a function is marked `override`, there is no point to also mark it `virtual`.